packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -304,6 +304,7 @@ class DraggableScrollableSheet extends StatefulWidget {
     this.snap = false,
     this.snapSizes,
     this.snapAnimationDuration,
+    this.snapAnimationCurve,
```

Comment:
Would you prefer a default value of `Curves.linear` here and make the parameter non nullable? I chose to do it this way to keep as much of the current implementation as possible but I think that would make the code cleaner since it would get rid of some conditional checks.

Reply:
I ended up going with a default value of Curves.linear here. Let me know if I should revert it.